config/m4/compiler.m4

```diff
@@ -1,12 +1,13 @@
 #
 # Copyright (C) Mellanox Technologies Ltd. 2001-2014.  ALL RIGHTS RESERVED.
+# Copyright (c) UT-Battelle, LLC. 2017. ALL RIGHTS RESERVED.
 # See file LICENSE for terms.
 #
 
 #
 # Initialize CFLAGS
 #
-CFLAGS="-g -Wall -Werror $UCX_CFLAGS"
+BASE_CFLAGS="-g -Wall -Werror $UCX_CFLAGS"
```

Comment:
UCX_CFLAGS is not defined anywhere - let's remove it
also, maybe we should define BASE_CPPFLAGS and BASE_CXXFLAGS here for better readability?